gamification/process/activity.py

```diff
@@ -0,0 +1,17 @@
+def get_activity(activity_type, labels):
+    """
+    Get an activity based on the labels on issues or mrs.
+
+    :param activity_type: a string representing the type of the
+                          activity. e.g. issue or merge_request.
+    :param labels:        a list of sorted labels.
+    :return:              a touple of points and activity string.
+    """
+    if activity_type == 'issue':
+        activity = 'Created a '
+    elif activity_type == 'merge_request':
+        activity = 'Solved a '
+    else:
+        raise UnboundLocalError('Activity is not defined.')
```

Comment:
`UnboundLocalError` is very wrong.  You need to define the variable `activity` before logic branches to avoid this exception.  Define it with an initial value of `None`